lib/watcher.js

```diff
@@ -61,6 +61,8 @@ function Watcher(logger, api, files, sources) {
 		this.busy = api.run(specificFiles || files, {
 			runOnlyExclusive: runOnlyExclusive
 		}).then(function (runStatus) {
+			runStatus.testRunElapsed = runStatus.testRunStart && process.hrtime(runStatus.testRunStart);
```

Comment:
Updated.

> That said I don't think there's much value in tracking test duration when in watch mode, especially since the number of test files run can vary greatly.

In that case, it's not very useful. When you're doing TDD or just updating a single test in general, it makes a lot more sense.